src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -630,7 +630,19 @@ void FancyZones::MoveWindowsOnDisplayChange() noexcept
 void FancyZones::UpdateDragState(require_write_lock) noexcept
 {
     const bool shift = GetAsyncKeyState(VK_SHIFT) & 0x8000;
-    m_dragEnabled = m_settings->GetSettings().shiftDrag ? shift : !shift;
+	const bool mouseR = GetAsyncKeyState(VK_RBUTTON) & 0x8000;

```

Comment:
When making changes to an existing file, please follow the existing code style/ formatting
In this case use 4 spaces instead of tab for indentation.
Thanks.

Reply:
I will make this change.